_includes/store/helsinkisystems-lexoffice.md

```diff
@@ -0,0 +1 @@
+{% include store/helsinkisystems-lexoffice.md %}
```

Comment:
This file must contain the entire product description.

Right now it references itself.